src/passwordcheck.js

```diff
@@ -16,7 +16,7 @@ define([
     options = options || {};
     var minLength = options.minLength || 8;
     var NO_OF_HASHING_FUNCTIONS = options.noOfHashFunctions || 8;
-    var bloom = new BloomFilter(BLOOM, NO_OF_HASHING_FUNCTIONS); //eslint-disable-line no-undef
+    var bloom = new BloomFilter(bloomdata, NO_OF_HASHING_FUNCTIONS); //eslint-disable-line no-undef
```

Comment:
Instead of using a global variable `BLOOM`, I added a snippet to plaintext2bloom [to add the RequireJS constructs](https://github.com/mozilla/fxa-password-strength-checker/pull/33/files#diff-3ab3fca7f1357d2d2d3a2c528da3e37bR33) to the generated files.